clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -137,6 +156,12 @@ class ServerService private (
 
 object ServerService {
 
+  /**
+    * Maximum number of updates that should be sent in a single bulk request
+    * to Elasticsearch during document recovery.
+    */
+  val RecoveryMaxBulkSize: Long = 100L
```

Comment:
We should probably up this number once we figure out why the integration tests aren't working